app/generator/content/schedule/part3/get-table.js

```diff
@@ -10,27 +12,29 @@ const getTable = (schedule) => {
     fillColor: '#d9d9d9',
     table: {
       headerRows: 1,
-      widths: ['*', ...schedule.map(x => ('*'))],
+      widths: ['*', '*', '*'],
       body: [
         [
           { text: 'Payment type', style: 'tableHeader' },
-          ...schedule.map(x => ({ text: x.paymentType, style: 'tableHeader' }))
-        ],
-        [
-          { text: 'Amount', style: 'tableHeader' },
-          ...schedule.map(x => ({ text: toCurrencyString(x.value), style: 'tableNumber' }))
-        ],
-        [
           { text: 'Payment period', style: 'tableHeader' },
-          ...schedule.map(x => ({ text: x.period, style: 'tableNumber' }))
-        ]
+          { text: 'Amount', style: 'tableHeader' }
+        ],
+        ...schedule.map(x => ([
+          { rowSpan: getRowSpan(x.paymentType, rowsCount, x.order), text: x.paymentType, style: 'tableNumber' },
```

Comment:
Need to see whether happy with `Quarterly payment` or whether they want it plural.

Reply:
Sample of generated schedule was presented to the team on 12/06/2023, everyone is happy with the content.